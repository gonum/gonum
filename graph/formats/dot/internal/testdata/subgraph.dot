digraph {
	{A}
	subgraph {B}
	subgraph S {C}
}
