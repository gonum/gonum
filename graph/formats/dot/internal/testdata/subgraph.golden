digraph {
	{A}
	{B}
	subgraph S {C}
}
